Shared/Samples/Choose camera controller/ChooseCameraControllerView.swift

```diff
@@ -0,0 +1,161 @@
+        surface.addElevationSource(elevationSource)
+        
+        // Sets the surface to the scene's base surface.
+        scene.baseSurface = surface
+        return scene
+    }()
+    
+    /// A graphics overlay containing an airplane graphic.
+    private static let graphicsOverlay: GraphicsOverlay = {
+        let graphicsOverlay = GraphicsOverlay(graphics: [Graphic.airplane])
+        graphicsOverlay.sceneProperties.surfacePlacement = .absolute
+        return graphicsOverlay
+    }()
+    
+    /// A Boolean value indicating whether the settings view should be presented.
+    @State var isShowingSettings = false
+    
+    /// The sun lighting mode of the scene view.
+    @State private var cameraControllerKind: CameraControllerKind = .globe
+    
+    enum CameraControllerKind: CaseIterable {
+        case globe
+        case plane
+        case crater
+        
+        /// A human-readable label for the camera controller kind.
+        var label: String {
+            switch self {
+            case .globe: return "Free pan round the globe"
+            case .plane: return "Orbit camera around plane"
+            case .crater: return "Orbit camera around crater"
+            }
+        }
+    }
+    
+    /// The camera controller of the scene view.
+    @State private var cameraController: CameraController {
+        didSet {
+            _ = GlobeCameraController()
+        }
+    }
+    
+    init() {
+        cameraController = Self.makeCameraController(kind: .crater)
+    }
+    
+    var body: some View {
+        VStack {
+            SceneView(
+                scene: scene,
+                cameraController: cameraController,
+                graphicsOverlays: [ChooseCameraControllerView.graphicsOverlay]
+            )
+            Menu("Camera Controllers") {
+                ForEach(CameraControllerKind.allCases, id: \.self) { kind in
+                    Button(kind.label) {
+                        cameraControllerKind = kind
+                    }
+                }
+            }
+            .onChange(of: cameraControllerKind) { newValue in
+                cameraController = Self.makeCameraController(kind: newValue)
+            }
+        }
+    }
+    
+    static func makeCameraController(kind: CameraControllerKind) -> CameraController {
+        switch kind {
+        case .crater:
+            let targetLocation = Point(
+                x: -109.929589,
+                y: 38.437304,
+                z: 1700,
+                spatialReference: .wgs84
+            )
+            let cameraController = OrbitLocationCameraController(
+                target: targetLocation,
+                distance: 5000
+            )
+            cameraController.cameraPitchOffset = 3
+            cameraController.cameraHeadingOffset = 150
+            return cameraController
+        case .plane:
+            guard let targetGraphic = graphicsOverlay.graphics.first else { return GlobeCameraController() }
```

Comment:
Then at here, because we know there is exactly 1 graphic on the overlay, we can force unwrap it.
```suggestion
            let targetGraphic = graphicsOverlay.graphics.first!
```